docs/HISTORY.txt

```diff
@@ -2,10 +2,13 @@ Changelog
 =========
 
 
-1.0.8 (unreleased)
+2.0.0 (unreleased)
```

Comment:
I would release it as `1.1.0`, not `2.0.0`, as it is just a backwards incompatible API change and not a complete rewrite 😉 
If you change it in `setup.py` too, `zest.releaser` will propose it correctly.